src/orders/views/OrderManualTransationRefund/OrderManualTransationRefund.tsx

```diff
@@ -0,0 +1,31 @@
+import { useOrderTransationsDataQuery } from "@dashboard/graphql";
+import { OrderManualTransationRefundPage } from "@dashboard/orders/components/OrderManualTransationRefundPage";
+import React from "react";
+
+import { filterRefundTransactions } from "./filter";
+
+interface OrderManualTransationRefundProps {
+  orderId: string;
+}
+
+const OrderManualTransationRefund = ({
+  orderId,
+}: OrderManualTransationRefundProps) => {
+  const { data, loading } = useOrderTransationsDataQuery({
+    displayLoader: true,
+    variables: {
+      orderId,
+    },
+  });
+
+  return (
+    <OrderManualTransationRefundPage
+      orderId={data?.order?.id ?? ""}
+      transactions={filterRefundTransactions(data?.order?.transactions ?? [])}
```

Comment:
I think transactions that are non-refundable should be displayed as disabled, instead of filtered out

Reply:
I'll do this in separate PR